### PR TITLE
Content Update

### PR DESCRIPTION
**Pull request by joinimran**
Added a source of information to point users to use custom settings for Windows 10 devices in Intune.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4482

### CODE REVIEW

**Comment by jvsam** [comment]
Thanks @joinimran. Can you please also correct the spelling of Integrity?

@JohanFreelancer9 @nenonix for copy review please. @illfated feel free to share your thoughts if you have time. Thank you!

File: windows/security/threat-protection/windows-defender-application-control/deploy-windows-defender-application-control-policies-using-intune.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ ms.date: 05/17/2018
 -   Windows 10
 -   Windows Server 2016
 
-You can use Microsoft Intune to configure Windows Defender Application Control (WDAC). You can configure Windows 10 client computers to only run Windows components and Microsoft Store apps, or let them also run reputable apps defined by the Intelligent Security Graph.   
+You can use Microsoft Intune to configure Windows Defender Application Control (WDAC). You can configure Endpoint protection profile for WDAC or a custom profile with an OMA-URI. You can configure Windows 10 client computers to only run Windows components and Microsoft Store apps, or let them also run reputable apps defined by the Intelligent Security Graph.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-control/deploy-windows-defender-application-control-policies-using-intune.md]:
```suggestion
You can use Microsoft Intune to configure Windows Defender Application Control (WDAC). You can configure the Endpoint protection profile for WDAC or a custom profile with an OMA-URI. You can configure Windows 10 client computers to only run Windows components and Microsoft Store apps, or let them also run reputable apps defined by the Intelligent Security Graph.   
```
I found it necessary to add the definite article "the" in front of "Endpoint protection profile" for syntax & grammar purposes in this sentence. I considered using "an" if it had been less specific, but I think "Endpoint protection profile" is specific enough to warrant using the definite article.

**Comment by illfated** [comment]
Nothing further to add from my side, but maybe @JohanFreelancer9 or @nenonix would like to suggest something more.

**Comment by joinimran** [comment]
Ready to go live.

**Comment by illfated** [comment]
#sign-off

**Comment by jvsam** [comment]
Thanks @Dansimp for merging this PR!

**Pull request merged**